File: JAVA/Prime_number.java
//prime number program
import java.util.Scanner;

public class Prime_number {

    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int num = input.nextInt();
        boolean value = true;
        if (num == 2) {
            System.err.println(num + " is a prime number");
        } else {
            for (int i = 2; i < num; i++) {
                if (num % i == 0) {
                    System.out.println(num + " is not a prime number");
                    value = false;
                    break;
                }
            }
        }
        if (value) {
            System.err.println(num + " is a prime number");
        }
    }

}

File: python/prime_number.py
num=int(input("Enter a number:- "))
if num<2:
 print("Not a prime number")
else: 
 for a in range (3,num):
    if num%2==0:
     print("Not a prime number")
     break
 else:
   print("It is a prime number")
